src/Configuration/Defaults/CommonSkippedRules.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Codestyle\Configuration\Defaults;
+
+use Blumilk\Codestyle\Configuration\SkippedRules;
+use PhpCsFixer\Fixer\ClassNotation\ClassAttributesSeparationFixer;
+use PhpCsFixer\Fixer\Operator\NotOperatorWithSuccessorSpaceFixer;
+use PhpCsFixer\Fixer\StringNotation\SingleQuoteFixer;
+
+class CommonSkippedRules implements SkippedRules
+{
+    protected array $rules = [
+        SingleQuoteFixer::class => null,
+        ClassAttributesSeparationFixer::class => null,
+        NotOperatorWithSuccessorSpaceFixer::class => null,
+    ];
```

Comment:
I would like to add 2 more fixers:
- `PhpCsFixer\Fixer\ReturnNotation\ReturnAssignmentFixer`
- `PhpCsFixer\Fixer\Operator\BinaryOperatorSpacesFixer`(temporary for union types)